### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by take2rohit**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -2,7 +2,7 @@
 Initiative to read research papers.
 
 ## Rules of appending papers:
-1. You should thoroughly read research paper and make proper notes using [https://hackmd.io/](HackMD).
+1. You should thoroughly read research paper and make proper notes using [HackMD](https://hackmd.io/).
 1. HackMD notes must mention those member who were part of reading group
 1. If notes and paper reading is complete append details in table
```
Comment (akshaykvnit) [inline README.md:7]: If notes and paper reading is completed, then update the details in the appropriate table below.

File: README.md [hunk 2]
```diff
@@ -2,7 +2,7 @@
...
 1. You can then create pull request to merge this
```
Comment (akshaykvnit) [inline README.md:8]: put a full stop at the end of all sentences

File: README.md (hunk 1 above)
Comment (take2rohit) [reply README.md]: Done

File: README.md (hunk 2 above)
Comment (take2rohit) [reply README.md]: done

**Pull request merged**